nuxt.config.ts

```diff
@@ -1,9 +1,9 @@
 // https://v3.nuxtjs.org/api/configuration/nuxt.config
 export default defineNuxtConfig({
-  shopware: {
-    shopwareEndpoint: process.env.API_URL,
-    shopwareAccessToken: process.env.API_ACCESS_TOKEN,
-  },
+  // shopware: {
```

Comment:
So this is using a static remote instance when there are no credentials provided? I think it's better this way because we don't need to mess around with the shopware instance at all. 👍